### PR TITLE
Project Plan Update: Tan Heng Yeow

### PR DESCRIPTION
**Pull request by tanhengyeow**
Hi @wkurniawan07 @tran-tien-dat 

I updated my project plan based on the discussion made in the past 2 weeks along with my progress update. Please review 😄

### CODE REVIEW

File: Tan_Heng_Yeow/progress.md [hunk 1]
```diff
@@ -22,20 +22,42 @@ Overall Direction:
...
-- [ ] Finish up the remaining work for this [PR](https://github.com/TEAMMATES/teammates/pull/7568)
+- [X] Finish up the remaining work for this [PR](https://github.com/TEAMMATES/teammates/pull/7568)
 - [ ] Write tests for features in the spreadsheet interface
 
+Works:
+- 1st review ready at https://github.com/TEAMMATES/teammates/pull/8833#issuecomment-389790326
+- Highlighted some thoughts on plan ahead at https://github.com/TEAMMATES/teammates/issues/8832#issuecomment-389788642
+
+Retrospect on Week 1:
+- Understanding what have been implemented so far
```
Comment (wkurniawan07) [inline Tan_Heng_Yeow/progress.md]: Retrospect is not a list of what you have done. It is more of a reflection on what you have done well, what could be improved, what are the difficulties faced, etc.

File: Tan_Heng_Yeow/progress.md [hunk 2]
```diff
@@ -44,6 +66,7 @@ Week 4 Plan:
 Work on feature "Updating email field of student through a context menu"
 
 - [ ] Finish up any remaining work from Week 3
+- [ ] Fix issues after code review
```
Comment (wkurniawan07) [inline Tan_Heng_Yeow/progress.md]: You don't really need this kind of entry.

**Review by wkurniawan07 — CHANGES_REQUESTED** [review]

**Review by wkurniawan07 — APPROVED** [review]
LGTM, @tran-tien-dat can merge if you're good with this

**Review by tran-tien-dat — APPROVED** [review]
LGTM

**Pull request merged**